packages/next/client/page-loader.js

```diff
@@ -72,8 +72,13 @@ export default class PageLoader {
     this.loadingRoutes = {
       // By default these 2 pages are being loaded in the initial html
       '/_app': true,
-      [initialPage]: true,
     }
+
+    // TODO get rid of this limitation for rendering the error page
```

Comment:
```suggestion
    // TODO: get rid of this limitation for rendering the error page
```

Highlighters and reporting tools rely on a colon